Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -1209,23 +1215,44 @@
         </ECCustomAttributes>
     </ECEntityClass>
 
+    <ECEntityClass typeName="ISurfaceControlElement" modifier="Abstract" displayLabel="Surface Control Element" description="An interface that can be mixed-into control objects and control rules to identify any entity that participates in surface control.">
+        <ECCustomAttributes>
```

Comment:
@Emi1yZhang , note that ISurfaceControlElement is supposed to be the base mix-in for ISurfaceControlObject. Once you introduce that change, you don't need to add ISurfaceControlElement as a mix-in for classes implementing ISurfaceControlObject (e.g. SurfaceControlVertex or Line).